releasenotes/notes/remove-tracer-attribute-from-span-9a765cf70003c9fe.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Removed tracer attribute from ``ddtrace.Span``. Accessing a tracer from a Span object is no longer supported.
```

Comment:
do we have an alternative that makes sense to include here?

Reply:
Not that I can think of. I think the key here is to avoid circular references